src/components/Form/InputRaw.js

```diff
@@ -99,22 +99,14 @@ const InputRaw = (props) => {
     return null;
   };
 
-  const renderSpinner = () => showSpinner && (
-    <Spinner
-      flavor="brand"
-      className="slds-input__spinner slds-m-right_xx-small"
-      size="x-small"
-    />
-  );
-
   const sldsClasses = [
     { 'slds-has-input-focus': isFocused },
     bare ? 'slds-input_bare' : 'slds-input',
     className
   ];
 
   return (
-    <span>
+    <Fragment>
```

Comment:
why not just use `<>` and `</>`

Reply:
Last time I checked, babel did not support that syntax yet without a specific plugin which I could not get to work in Storybook. This might be outdated now though

Reply:
@quadreex I don't think there is a way around this since we are still using Babel 6 in this project and JSX Fragment support starts with a late Babel 7 beta. I'd prefer to merge this for now and do the whole storybook / babel update at a later date when storybook support for babel 7 and webpack 4 is [out of beta](https://github.com/storybooks/storybook/issues/3805)

Reply:
👍